packages/mongo-menu/src/MongoMenu.spec.tsx

```diff
@@ -57,4 +65,24 @@ describe('packages/MongoMenu', () => {
 
     expect(onLogout).toHaveBeenCalledTimes(1);
   });
+
+  test('renders the account link as a disabled button when set to the empty string', () => {
+    renderedComponent.rerender(
```

Comment:
I've never used Jest or this testing library before, so **please** feel free to correct any style things I may have done incorrectly here.